redbot/cogs/mutes/mutes.py

```diff
@@ -1532,6 +1532,15 @@ async def mute_user(
             if guild.id not in self._server_mutes:
                 self._server_mutes[guild.id] = {}
 
+            # This is here in the event that the author tries to update the mute duration of user.
+            # we must check if there is an automatic unmute task in the self._server_mutes cache, for the specific guild.id and user.id.
+            # if there is, then we cancel the automatic unmute task.
+            if user.id in self._server_mutes[guild.id]:
+                task_name = f"server-unmute-{guild.id}-{user.id}"
+                print(self._unmute_tasks)
```

Comment:
This file has logging setup. So if you really want to keep this here I recommend changing to `log.debug` rather than print or remove this entirely. It might be worth considering doing a `log.info(f"{user} has had their mute time updated in {guild}")` but I don't think that's entirely necessary.

Reply:
> This file has logging setup. So if you really want to keep this here I recommend changing to `log.debug` rather than print or remove this entirely. It might be worth considering doing a `log.info(f"{user} has had their mute time updated in {guild}")` but I don't think that's entirely necessary.

Oh sorry that was just a sanity check for me, I'll delete it.